docs/12.EXT-REFERENCE-MODULE.md

```diff
@@ -130,20 +163,36 @@ load_and_evaluate_js_file (const jerry_char_t *name, jerry_value_t *result)
 
   return return_value;
 }
+
+static jerry_value_t
+canonicalize_file_path (const jerry_value_t name)
+{
+  jerry_value_t resulting_path;
+
+  /* Search name for "./" and "../", follow symlinks, etc., then create resulting_path via jerry_create_string () */
+
+  return resulting_path;
+}
+
+static jerryx_module_resolver_t js_file_loader
+{
+  canonicalize_file_path,
+  load_and_evaluate_js_file
+};
 ```
 
 We can now load JavaScript files:
 ```c
 static const jerryx_module_resolver_t resolvers[] =
 {
   /* Consult the JerryScript module resolver first, in case the requested module is a compiled-in JerryScript module. */
-  jerryx_module_native_resolver,
+  &jerryx_module_native_resolver,
```

Comment:
Why do you need & ?

Reply:
Because `jerryx_module_native_resolver` is now a structure stored in a global variable, and this is an array of structure pointers, so I will update the declaration of `resolvers` a few lines up.